sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateStore.scala

```diff
@@ -135,7 +135,14 @@ trait StateStore extends ReadStateStore {
       valueSchema: StructType,
       keyStateEncoderSpec: KeyStateEncoderSpec,
       useMultipleValuesPerKey: Boolean = false,
-      isInternal: Boolean = false): Short
+      isInternal: Boolean = false): Unit
+
+  // Test-visible methods to fetch column family mapping for this State Store version
```

Comment:
Shall we put this into RocksDBStateStore and in test we perform explicit type conversion to RocksDBStateStore? This still put the concept of column family ID into top level of StateStore interface.

Reply:
Okay sure